src/Kongverge/Program.cs

```diff
@@ -107,6 +111,11 @@ public Options(CommandLineApplication app)
                     return ExitCode.IncompatibleArguments;
                 }
 
+                if (!BasicAuthUser.HasValue() && BasicAuthPassword.HasValue())

```

Comment:
This should also check for if the user has been provided but not the password.

Reply:
The password can be read via stdin if it's not provided as a parameter.  This ensures it's more secure as it won't show in any logs.

Reply:
I know that. That's why you need to check both places here, rather than in the `Apply` method.